packages/report-server/src/__tests__/reportBuilder/context/buildContext.test.ts

```diff
@@ -44,14 +43,12 @@ describe('buildContext', () => {
     ],
   };
 
-  const apiMock = entityApiMock(ENTITIES, RELATIONS);
-
   const reqContext: ReqContext = {
     hierarchy: HIERARCHY,
     permissionGroup: 'Public',
-    services: {
-      entity: apiMock,
-    } as ReqContext['services'],
+    services: new MockTupaiaApiClient({
+      entity: new MockEntityApi(ENTITIES, RELATIONS),
+    }),
```

Comment:
I'm pretty happy with how elegant this pattern turned out 😄 and not having to do explicit type casting anymore is a nice bonus imo!